controllers/ixiatg_controller.go

```diff
@@ -55,6 +55,11 @@ const (
 	CURRENT_NAMESPACE string = "ixiatg-op-system"
 	SECRET_NAME       string = "ixia-pull-secret"
 
+	LIC_ADDR_SECRET   string = "license-server"
+	LIC_SERVER_SECRET string = "license-server-image"
+	LIC_SERVER_PATH   string = "ghcr.io/open-traffic-generator/licensed/keng-license-server"
```

Comment:
Can we avoid hard-coding this path? If we do it this way it visible to everyone.